platform/windows/display_server_windows.cpp

```diff
@@ -4742,7 +4781,7 @@ DisplayServer::WindowID DisplayServerWindows::_create_window(WindowMode p_mode,
 	DWORD dwExStyle;
 	DWORD dwStyle;
 
-	_get_window_style(window_id_counter == MAIN_WINDOW_ID, (p_mode == WINDOW_MODE_FULLSCREEN || p_mode == WINDOW_MODE_EXCLUSIVE_FULLSCREEN), p_mode != WINDOW_MODE_EXCLUSIVE_FULLSCREEN, p_flags & WINDOW_FLAG_BORDERLESS_BIT, !(p_flags & WINDOW_FLAG_RESIZE_DISABLED_BIT), p_mode == WINDOW_MODE_MAXIMIZED, (p_flags & WINDOW_FLAG_NO_FOCUS_BIT) | (p_flags & WINDOW_FLAG_POPUP), dwStyle, dwExStyle);
+	_get_window_style(window_id_counter == MAIN_WINDOW_ID, (p_mode == WINDOW_MODE_FULLSCREEN || p_mode == WINDOW_MODE_EXCLUSIVE_FULLSCREEN), p_mode != WINDOW_MODE_EXCLUSIVE_FULLSCREEN, p_flags & WINDOW_FLAG_BORDERLESS_BIT, !(p_flags & WINDOW_FLAG_RESIZE_DISABLED_BIT), p_mode == WINDOW_MODE_MAXIMIZED, false, (p_flags & WINDOW_FLAG_NO_FOCUS_BIT) | (p_flags & WINDOW_FLAG_POPUP), dwStyle, dwExStyle);
```

Comment:
`maximized_fs` is handled inside `WM_WINDOWPOSCHANGED` [This is required because maximization could occur through external means (via the menu) which bypasses our state management logic at `window_set_mode`👍].

In case needed the following can be used instead here of `false` (not needed for now because it is handled at `WM_WINDOWPOSCHANGED`):
```c++
bool is_maximized_fs = (p_mode == WINDOW_MODE_MAXIMIZED) && (p_flags & WINDOW_FLAG_BORDERLESS_BIT) && (screen_get_size(rq_screen) == screen_get_usable_rect(rq_screen).size) && (screen_get_position(rq_screen) == screen_get_usable_rect(rq_screen).position);
```